src/dos/cdrom.h

```diff
@@ -154,37 +155,51 @@ class CDROM_Interface_Image : public CDROM_Interface
 {
 private:
 	class TrackFile {
+	protected:
+		TrackFile(Bit16u chunkSize) : chunkSize(chunkSize) {}
 	public:
-		virtual bool read(Bit8u *buffer, int seek, int count) = 0;
-		virtual int getLength() = 0;
-		virtual ~TrackFile() { };
+		virtual bool   read(Bit8u *buffer, int seek, int count) = 0;
+		virtual bool   seek(Bit32u offset) = 0;
+		virtual Bit16u decode(Bit8u *buffer) = 0;
+		virtual Bit16u getEndian() = 0;
+		virtual Bit32u getRate() = 0;
+		virtual Bit8u  getChannels() = 0;
+		virtual int    getLength() = 0;
+		virtual        ~TrackFile() { };
```

Comment:
Excellent; I prefer `default`over `{ }` because the later might imply that nothing is destructed as opposed to tearing down member objects and baseclasses (which `default` does). 
Actually, would it make more sense for me to delete this destructor all together - which implies default?

Reply:
Fixed in 66fdb679